10850 pinconstructor

VTsarevNSU opened the pull request.

VTsarevNSU committed:
промежуточный коммит

коммит перед мерджем дева в эту ветку
основной функционал почти готов, но здесь много TODO, они будут закрыты после мёрджа, поскольку нужна база данных и навигация

VTsarevNSU committed:
небольшой багфикс

теперь всё запускается

VTsarevNSU committed:
Merge branch 'dev' into 10850-pinconstructor

DoHKapJleoHe reviewed (approved):
pic_example.jpg смешной. посмеялся)

VTsarevNSU committed:
багфикс

теперь код можно попытаться запустить, чтобы он упал

VTsarevNSU committed:
Merge pull request #10 from NSU-PinMap/dev

10850-pinconstructor

VTsarevNSU committed:
сделал нормальный вызов фрагмента

по нажатию на маркер вызывается конструктор, в который передались данные из пина
